wondrous-app/components/Icons/walletSetupModalIcons.tsx

```diff
@@ -0,0 +1,255 @@
+export const DefaultWalletTypeIcon = ({ bgFill = '#232323', iconFill = '#7A7A7A' }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="22" height="23" fill="none" viewBox="0 0 22 23">
+    <rect width="22" height="22" y="0.935" fill={bgFill} rx="4" />
+    <path
+      fill={iconFill}
+      d="M6.875 6.435c-.72 0-1.312.567-1.365 1.274a.458.458 0 00-.01.1v8.251c0 .754.621 1.375 1.375 1.375h8.25c.754 0 1.375-.621 1.375-1.375V9.643c0-.754-.621-1.375-1.375-1.375h-8.25a.451.451 0 01-.458-.458c0-.259.2-.459.458-.459h7.639a.459.459 0 100-.916H6.875zm-.458 2.75h8.708c.259 0 .458.2.458.458v6.417c0 .258-.2.458-.458.458h-8.25a.451.451 0 01-.458-.458V9.185zm7.486 2.75a.764.764 0 100 1.528.764.764 0 000-1.528z"
+    />
+  </svg>
+);
+
+export const GnosisWalletTypeIcon = ({ fill = '#008C73' }) => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="none" viewBox="0 0 20 20">
+    <path
+      fill={fill}
+      d="M10 0C4.473 0 0 4.48 0 10c0 5.527 4.48 10 10 10s10-4.48 10-10S15.527 0 10 0zm7.32 10.546h-4.64a2.83 2.83 0 01-3.323 2.224 2.83 2.83 0 01-2.223-3.323 2.83 2.83 0 013.322-2.223A2.83 2.83 0 0112.7 9.563h4.621c.27-.013.501.2.508.476a.486.486 0 01-.476.507h-.032z"
+    />
+  </svg>
+);
+
+export const MetamaskWalletTypeIcon = () => (
```

Comment:
can we use the existing Metamask  icon `components/Icons/metamask.tsx`? if not this is fine

Reply:
certainly